docs/application/native/guides/internationalization/i18n.md

```diff
@@ -872,6 +876,63 @@ using a Number format for the PluralFormat locale.
 To add custom plural rules to the PluralFormat, create a Plural Rules object and add it in PluralFormat constructor.
 If you also pass a locale object to constructor, this locale will affect formatted number in the returned message texts.
 
+<a name="date_interval"></a>
+## Usages of date intervals
+
```

Comment:
I tried to avoid two heading one after other and paragraph about creating date intervals is short. If there is a need to add this heading, please tell me and I will add it in next commit.

Reply:
I agree with your point. But to maintain consistency, it is better to add headings.